decidim-core/app/controllers/decidim/account_controller.rb

```diff
@@ -56,6 +56,72 @@ def destroy
       redirect_to decidim.root_path
     end
 
+    def resend_confirmation_instructions
+      enforce_permission_to :update, :user, current_user: current_user
+
+      ResendConfirmationInstructions.call(current_user) do
+        on(:ok) do
+          respond_to do |format|
+            format.js do
+              render(
+                "resend_confirmation_email",
```

Comment:
I think we should:
1. Change the `js.erb` template name the same as the method name here (`resend_confirmation_instructions`)
2. Instead of passing "locals" to the method, just use instance variables (`@text`, `@alert_class`)

Same also for the invalid block.